citadel/sensors.md

```diff
@@ -131,7 +131,7 @@ Now we can add the `contact` sensor to the `box` link of the `wall` model:
 
 The definition of the `<sensor>` is straight forward, we just define the `name` and the `type` of the sensor. And inside the `collision` we define the box link collision name which is `collision`.
 
-We need also to add the `TouchPlugin` under the `wall` model as follows:
+We need also to add the `TouchPlugin` under the link in the `wall` model as follows:
```

Comment:
I can see how the original wording can be confusing, but I think that the suggested wording may have people putting the plugin inside `<link>`. How about:

```suggestion
We need also to add the `TouchPlugin` inside the `wall` model as follows:
```

Reply:
Ping @isabelleysseric , what do you think?

Reply:
@isabelleysseric , I see you marked this comment as resolved, but you haven't addressed. Do you agree with the suggestion?